content/200-orm/050-overview/500-databases/250-prisma-postgres.mdx

```diff
@@ -98,6 +98,10 @@ It's not yet possible to configure the size and resource allocation of your Pris
 
 Prisma Postgres will be gradually rolled out in new regions. If it is not yet available in your preferred region, you can simply subscribe to get notified when it becomes available by subscribing to notifications in the **Region** dropdown when setting up your Prisma Postgres instance in the Platform Console. We'll send you an email when new regions become available, so you can easily stay up to date.
 
+### Fixed connection limit
+
+Prisma Postgres currently has a fixed connection limit of 10, which cannot be increased. This limitation is temporary and applies only while Prisma Postgres is in Early Access. Once Prisma Postgres reaches General Availability (GA), it will support configurable connection limits, allowing you to scale connections as needed.
```

Comment:
```suggestion
Prisma Postgres currently has a fixed connection limit of 10 that cannot be adjusted. This is a temporary limitation during Early Access. Once Prisma Postgres reaches General Availability (GA), it will support configurable connection limits, enabling you to scale connections according to your needs.
```